apps/web/components/dashboard/bookmarks/EditorCard.tsx

```diff
@@ -139,31 +138,33 @@ export default function EditorCard({ className }: { className?: string }) {
     }
   };
 
+  const OS = getOS();
+
   return (
     <Form {...form}>
       <form
         className={cn(
           className,
-          "flex flex-col gap-2 rounded-xl bg-card p-4",
+          "relative flex flex-col gap-2 rounded-xl bg-card p-4",
           cardHeight,
         )}
         onSubmit={form.handleSubmit(onSubmit, onError)}
       >
-        <div className="flex justify-between">
-          <p className="text-sm">NEW ITEM</p>
```

Comment:
Why though? I didn't have this before, and I added it because the editor card wasn't very distinguishable from the rest of the cards.

Reply:
Reverted this change for now. Let's discuss and pick this separately as UI/UX improvement